modules/service/src/main/resources/db/migration/V0863__call_for_proposals_update.sql

```diff
@@ -0,0 +1,33 @@
+DROP VIEW v_cfp;
+
+ALTER TABLE t_cfp
+  ADD c_active_start timestamp,
+  ADD c_active_end   timestamp;
+
+UPDATE t_cfp
+  SET c_active_start = COALESCE(lower(c_active), '2000-01-01 00:00:00'::timestamp),
+      c_active_end   = COALESCE(upper(c_active), '2000-01-01 00:00:00'::timestamp);
+
+ALTER TABLE t_cfp
+  DROP c_active,
```

Comment:
I switched this from a `tsrange` to two `timestamp` columns because I couldn't figure out how to deal with it in Grackle.